script/receiver

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
+
```

Comment:
I'm good with executables living in `bin` as long as the names are qualified, e.g. `travis-logs-drain`.  It's a highly subjective topic 😁